src/ui/Controller.java

```diff
@@ -1,21 +1,23 @@
 package ui;
 
 import data.DataParser;
+import data.EmployeeRepository;
 import data.OnEventListener;
+import model.Employee;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class Controller implements OnEventListener {
     private final DataParser dataParser = new DataParser();
 
```

Comment:
Add method for input (id, name)  which searches for Employee with id and name in the file